wit_dashboard/wit-dashboard.html

```diff
@@ -3675,15 +3739,30 @@ <h2>Create similarity feature</h2>
           // - A list of numbers representing the attributions to individual
           //   values of a multivalent feature - such as pixels in an image or
           //   numbers in a list of numbers.
-          // - A 2D list for sparse attributions of a multivalent feature. The
-          //   first list contains the feature values for which attributions
-          //   exist. The second list contains the numeric attributions for each
-          //   of those feature values.
           attributions: {
             type: Object,
             value: {indices: [], attributions: []},
             observer: 'newAttributions_',
           },
+          // Stores all attributions for all datapoints for each model, for
+          // use in generating global attribution stats.
+          allAttributions_: {
+            type: Array,
+            value: () => [],
+          },
+          // Stores for each model the list of features and their mean
+          // attributions.
+          meanAttributions_: {
+            type: Array,
+            value: () => [],
+          },
+          // When faceting the dataset for performance evaluation, stores for
```

Comment:
can you add one example indexing for these structures (e.g. allAttributions:  modelInd -> featureInd)

Reply:
added detailed type documentation to allAttributions, meanAttributions and facetedMeanAttributions